oriedita-ui/src/main/java/oriedita/editor/swing/LeftPanel.java

```diff
@@ -244,6 +245,7 @@ public LeftPanel(@Named("mainFrame") JFrame frame,
         buttonService.registerButton(correctCpBeforeFoldingCheckBox, "correctCpBeforeFoldingAction");
         buttonService.registerButton(selectPersistentCheckBox, "selectPersistentAction");
         buttonService.registerButton(coloredXRayCheckBox, "coloredXRayAction");
+        buttonService.registerButton(drawDiagonalGridlinesCheckBox, "drawDiagonalGridlinesAction");
```

Comment:
In the `.properties` you call this `drawDiagonalGridLinesAction`, with a capital L

Reply:
thank you! so silly :S. Fixed now 🥳